content/api/vision/_index.md

```diff
@@ -19,7 +19,12 @@ The markers in the list have some useful attributes:
 - `id` - returns the [id](marker-ids) of the marker.
 - `pixel_centre` - returns the location in pixels of the centre of the marker in the captured image.
 - `cartesian` - returns details of the position of the marker in the [Cartesian](coordinates/#cartesian-coordinates) coordinate system.
-- `polar` - *Deprecated*. Returns coordinate data in an undocumented manner.
+- `spherical` - returns details of the position of the marker in a [spherical](coordinates/#spherical-coordinates) coordinate system.
+- `polar` - *Deprecated*. Returns coordinate data in an undocumented manner. Use the `spherical` attribute instead.
```

Comment:
> Returns coordinate data in an undocumented manner

Perhaps we should remove this? It's unnecessary information, and portrays us in slightly bad light

Reply:
a5d627b